moveit_planners/pilz_industrial_motion_planner/doc/MotionBlendAlgorithmDescription.fls

```diff
@@ -0,0 +1,147 @@
+PWD /home/ch/ros/pilz_ws/src/moveit/moveit_planners/pilz_industrial_motion_planner/doc
```

Comment:
I agree with Dave that the source repo should only comprise source files. Please remove all tex output from the commit history and just keep .tex and image sources.
Particularly, remove:
*.fdb_latexmk
*.fls
*.out
*.synctex.gz
MotionBlendAlgorithmDescription.pdf (created from .tex)
.png and .pdf that are generated from .uxf (at least keep only one of the generated files)
ptp.png (generated from ptp.svg)
*-eps-converted-to.pdf (I suggest to keep pdfs instead of .eps files)

Also, do we really need the email confirmations from @clalancette and @gavanderhoorn for changing license terms in this repo? Isn't it sufficient that you can provide them if necessary?

Reply:
@rhaschke @gavanderhoorn I'm happy to review new licensing terms, but I think I'm missing some context.  Can you please point me to what the license was and what it will be in the future?

Reply:
@clalancette the change was from LVGLv3 to BSD 3-clause. But we talked about it already, as documented here: https://github.com/PilzDE/moveit/blob/pilz_planner_merge/moveit_planners/pilz_industrial_motion_planner/doc/E-Mail_clalancette.pdf 
(which maybe shows to @rhaschke why it may be handy to have the files around ;-) )

Reply:
@rhaschke I do agree with you not to check in the latex auxiliary files. But I would like to check in the final pdf, to be able to make it available directly for documentation purposes. I can not expect from someone looking for information on the planner to first build the tex files.

Reply:
The preferred format for documentation in MoveIt is via sphinx in the tutorials. Is there a possibility to convert/compile latex into .rst files?

Reply:
I have tried to convert the file with [pandoc](https://pandoc.org/). But it does not handle math and graphics very well.
Where would you put this documentation anyways? Under https://moveit.ros.org/documentation/planners/ I assume. Because all deeper links from there seem to pretty inconsistent anyways.

Reply:
I was thinking of the MoveIt tutorials. But let's keep this separated:
- We should definitely have a tutorial how to use this planner - probably you already have this.
- In https://moveit.ros.org/documentation/planners we should have a short summary of functionality and there you could link your pdf.

Reply:
Yes, this was also what I was thinking. 

@jschleicher created the tutorial PR: https://github.com/ros-planning/moveit_tutorials/pull/461

And I will then focus on the https://moveit.ros.org/documentation/planners part